lib/plenario_web/controllers/web/page_controller.ex

```diff
@@ -19,8 +19,15 @@ defmodule PlenarioWeb.Web.PageController do
 
     {startdt, conn} = parse_date(conn, starts)
     {enddt, conn} = parse_date(conn, ends)
-
-    do_explorer(zoom, coords, startdt, enddt, conn)
+    comparison = DateTime.compare(startdt, enddt)
+
+    case DateTime.compare(startdt, enddt) do
+      :gt ->
+        do_explorer(zoom, coords, startdt, enddt, put_flash(conn, :error,
+          "The starting datetime #{startdt} cannot be greater than the ending datetime #{enddt}"))
```

Comment:
While we're in here making changes, perhaps it's worth applying a formatterto the date display in the flash. This doesn't exactly scream user-friendly, and they'll never have time components anyway:

> The starting datetime 2018-05-16 00:00:00.0Z cannot be greater than the ending datetime 2018-05-16 00:00:00.0Z